source/documentation/appendix-docs/botor.md

```diff
@@ -47,18 +65,23 @@ Then in the **terminal** run
 ```bash
 python3 -m venv renv/venv --without-pip --system-site-packages
 ```
+to create a Python virtual environment.
 
 Finally, in the RStudio console run the remaining lines:
 
 ```r
+# Initialise the renv environment. Remove bare = TRUE if you'd like to move 
+# your existing packages over to renv, but keep it set to TRUE if you're 
+# already using botor, dbtools or reticulate otherwise it will point to the 
+# wrong or a non-existent Python.
```

Comment:
Do we want these notes here, or in the above cell where we initialise `renv`?

Reply:
Corrected - b1a8edc